drivers/storage/portworx/portworx.go

```diff
@@ -310,29 +310,33 @@ func (p *portworx) preflightShouldRun(toUpdate *corev1.StorageCluster) bool {
 		return check == "true"
 	}
 
-	if !preflight.RequiresCheck() { // Preflight is only supported on AWS, VSPHERE & Pure
+	/*if !preflight.RequiresCheck() { // Preflight is only supported on AWS, VSPHERE & Pure
```

Comment:
Let's remove the commented code. We can always revert this commit to get the deleted changes.